### PR TITLE
 set-volume, enable-pnpdevice, get-dedupproperties: add pages

### PR DESCRIPTION
**Pull request by kabir1439**
[enable-pnpdevice.md](https://github.com/tldr-pages/tldr/files/15434998/enable-pnpdevice.md)
[get-dedupproperties.md](https://github.com/tldr-pages/tldr/files/15434999/get-dedupproperties.md)
[set-volume.md](https://github.com/tldr-pages/tldr/files/15435000/set-volume.md)

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Welcome to tldr and thanks for your contribution. Can you sign the CLA in the above comment for this PR to go through?

**Comment by spageektti** [comment]
Hi. You dont have to open a new PR every change you make(#12829 #12830).

**Comment by spageektti** [comment]
Isn't the `Set-Volume.md` page duplicate of `set-volume.md`? Also, as I said on the first PR, the command should probably be in the windows directory like all other Powershell commands.

File: pages/windows/enable-pnpdevice.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Enable-PnpDevice
+
+> The Enable-PnpDevice cmdlet enables a Plug and Play (PnP) device. You must use an Administrator account to enable a device.
+> This commnand can only be used through Powershell.
```
[the remark this answers is not in the document]
Comment (spageektti) [reply pages/windows/enable-pnpdevice.md:4]: Are you sure about the `Note:`?
```sh
grep -r "Note: This command can only be used through PowerShell." pages/windows/ | wc -l
0
```
```sh
grep -r "This command can only be used through PowerShell." pages/windows/ | wc -l
29
```

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/windows/enable-pnpdevice.md:4]: > Are you sure about the Note:?

Those pages should probably be updated for the same (will do it).

File: pages/windows/enable-pnpdevice.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+`Enable-PnpDevice -InstanceId 'RETRIEVED USING Get-PnpDevice COMMAND'`
+
+- Enable all disabled Pnp devices:
+`Get-PnpDevice | Where-Object {$_.Problem -eq 22} | Enable-PnpDevice`
+
+- Enable a device without confirmation:
+`Enable-PnpDevice -InstanceId 'RETRIEVED USING Get-PnpDevice COMMAND' -Confirm:$False`
+
+- Shows what would happen if the cmdlet runs. The cmdlet is not run:
+`Enable-PnpDevice -InstanceId 'USB\VID_5986&;PID_0266&;MI_00\7&;1E5D3568&;0&;0000' -WhatIf:$True`
```
Comment (kbdharun) [inline pages/windows/enable-pnpdevice.md:17]:
```suggestion
- Enable a device:

`Enable-PnpDevice -InstanceId 'RETRIEVED USING Get-PnpDevice COMMAND'`

- Enable all disabled Pnp devices:

`Get-PnpDevice | Where-Object {$_.Problem -eq 22} | Enable-PnpDevice`

- Enable a device without confirmation:

`Enable-PnpDevice -InstanceId 'RETRIEVED USING Get-PnpDevice COMMAND' -Confirm:$False`

- Dry run of what would happen if the cmdlet runs:

`Enable-PnpDevice -InstanceId 'USB\VID_5986&;PID_0266&;MI_00\7&;1E5D3568&;0&;0000' -WhatIf:$True`
```

File: pages/windows/get-dedupproperties.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+> Gets Data Deduplication information
+> This command can only be used through Powershell.
+> More Information: <https://learn.microsoft.com/powershell/module/storage/get-dedupproperties>.
+
+- Get Data Deduplication information of the Drive:
+`Get-DedupProperties -DriveLetter 'C'`
+
+- Get Data Deduplication information of the Drive using the drive label:
+`Get-DedupProperties -FileSystemLabel 'Label'`
```
Comment (kbdharun) [inline pages/windows/get-dedupproperties.md:11]:
```suggestion
- Get Data Deduplication information of the Drive:

`Get-DedupProperties -DriveLetter 'C'`

- Get Data Deduplication information of the Drive using the drive label:

`Get-DedupProperties -FileSystemLabel 'Label'`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have resolved the Unix line ending (LF) issue raised by the linter along with the suggested fixes (since it's your first time contributing to tldr).

---

In future, I would suggest enabling automatic line ending resolution in Git using the command `git config --global core.autocrlf true` or if you are using an IDE like VSCode set the same over there too.

By default, Windows uses CRLF as line endings, whereas UNIX platforms like macOS and Linux use LF (unix-style line endings). We use LF as the default line ending at `tldr`.

**Review by spageektti — APPROVED** [review]

**Pull request merged**